runtime/debug/debug.go

```diff
@@ -351,7 +348,6 @@ func Setup(ctx *cli.Context) error {
 }
 
 func startPProf(address string) {
-	http.Handle("/memsize/", http.StripPrefix("/memsize", &Memsize))
```

Comment:
Should add a note in changelog that this has been removed, I think this would be considered a breaking change , correct @prestonvanloon ?

We can no longer access `/memsize` from the profiling port

Reply:
I removed it here: https://github.com/prysmaticlabs/prysm/pull/14351/commits/51935959d63de4e00a17ed0be3a40c1ece3a01ef

 because of this: https://github.com/fjl/memsize/issues/5 . With go 1.23, this would be broken. Geth has also removed it on their end:
 https://github.com/ethereum/go-ethereum/commit/e4675771eda550e7eeb63a8884816982c1980644